rustls/tests/common/mod.rs

```diff
@@ -310,6 +325,13 @@ impl KeyType {
             .collect()
     }
 
+    pub fn get_spki(&self) -> SubjectPublicKeyInfoDer<'static> {
+        rustls_pemfile::public_keys(&mut io::BufReader::new(self.bytes_for("end.spki.pem")))
+            .map(|result| result.unwrap())
+            .next() // We only expect one RPK.
+            .unwrap()
```

Comment:
Apologies, main has moved under you a bit. This can be replaced with:

```suggestion
        SubjectPublicKeyInfoDer::from_pem_slice(self.bytes_for("end.spki.pem"))
            .unwrap()
```
(ditto below)

Reply:
Yup, I noticed after the CI crashing. Was just updating it. Pretty cool pattern to load the pemfiles I think.